packages/devtools_flutter/test/wrappers.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+/// Wraps [widget] with the build context it needs to load in a test.
+Widget wrap(Widget widget, {Size windowSize}) => MaterialApp(home: widget);
```

Comment:
nit: remove `windowSize` param if it is not used, or instead use it in the return somehow

Reply:
That's a good catch.  I ended up needing a separate windowSize setter that I added below.  Done.